src/roman_datamodels/testing/factories.py

```diff
@@ -143,6 +144,23 @@ def _random_array_uint32(size=(4096, 4096), min=None, max=None):
     return np.random.randint(min, high=max, size=size, dtype=np.uint32)
 
 
+def _random_dq_def(size=10, dq_size=32):
+    table = np.zeros((size,), dtype=tables.DQ_DEF_DTYPE)
```

Comment:
Rename to dq_table? Again, for ease of reading

Reply:
Sure, I'll call this dq_def since that seems to be the name it's know by